apps/public-docsite/package.json

```diff
@@ -1,12 +1,14 @@
 {
   "name": "@fluentui/public-docsite",
   "version": "1.0.0",
-  "private": true,
```

Comment:
we need to add below to ensure beachball generates correct change type for this package
```
  "beachball": {
    "tag": "latest",
    "disallowedChangeTypes": [
      "major",
      "prerelease"
    ]
  },
```

Reply:
Reason being we have [this](https://github.com/microsoft/fluentui/blob/master/scripts/beachball/index.ts#L4) as global/default config